ekf_se_odom:
  use_sim_time: false
  frequency: 50.0
  two_d_mode: true
  publish_tf: true

  map_frame: map
  odom_frame: odom
  base_link_frame: base_link
  world_frame: odom

  odom0: /wheel_odom
  odom0_config: [true,  true,  false,
                 false, false, true,
                 true,  true,  false,
                 false, false, true,
                 false, false, false]
  odom0_differential: false
  odom0_relative:     false

  imu0: /imu/data_centered
  imu0_config: [false, false, false,
                false, false, true,
                false, false, false,
                false, false, true,
                false, false, false]
  imu0_differential: false
  imu0_relative:     false
  imu0_remove_gravitational_acceleration: true
